packages/slice-machine/lib/models/server/ServerState.ts

```diff
@@ -16,15 +16,17 @@ export interface ConfigErrors {
 export interface ServerState {
   libraries: ReadonlyArray<Library>;
   customTypes: ReadonlyArray<CustomType<ObjectTabs>>;
+  remoteCustomTypes: ReadonlyArray<CustomType<ObjectTabs>>;
   remoteSlices: ReadonlyArray<Slice<AsObject>>;
-  clientError?: ErrorWithStatus;
-  configErrors: ConfigErrors;
-  env: Environment;
+  clientError: ErrorWithStatus | undefined;
+  configErrors: ConfigErrors | undefined;
```

Comment:
the type changes, are you sure this is right ?
I never saw configErrors as optional, I thin I saw it's an object wth keys, so i think it can be an empty object.

Reply:
Client error are coming from this function : 

```
export default async function handler(env: BackendEnvironment): Promise<{
  remoteSlices: ReadonlyArray<Slice<AsObject>>;
  clientError: ErrorWithStatus | undefined;
  libraries: ReadonlyArray<Library>;
}>
```

So before the type doesn't match well the reality. 


For the config errors, the getEnv return type told that the function can return `configErrors: ConfigErrors | undefined;` but this is not the case the reality. So I changed the returnType and force the serverState to `configErrors: ConfigErrors` ✅